### PR TITLE
BAM-73 Added a task to allow add tags to EC2 instances and minor fix …

### PR DESCRIPTION
**Pull request by spalter**
Hey, @jzillmann may you can have a look into the changes? Minor fix for the region changes from our latest change and I've added a EC2 Task to tag running instances.

### CODE REVIEW

File: src/main/java/datameer/awstasks/util/Ec2Configuration.java [hunk 1]
```diff
@@ -27,12 +27,15 @@
...
 import awstasks.com.amazonaws.auth.BasicAWSCredentials;
+import awstasks.com.amazonaws.regions.Region;
+import awstasks.com.amazonaws.regions.Regions;
 import awstasks.com.amazonaws.services.ec2.AmazonEC2;
 import awstasks.com.amazonaws.services.ec2.AmazonEC2Client;
 import awstasks.com.amazonaws.services.ec2.model.RunInstancesRequest;
 import awstasks.com.amazonaws.services.s3.AmazonS3;
 import awstasks.com.amazonaws.services.s3.AmazonS3Client;
 
+
```
Comment (lemaxm) [inline src/main/java/datameer/awstasks/util/Ec2Configuration.java:38]: Try to omit unnecessary new lines

**Comment by jzillmann** [comment]
Looks good!

File: src/main/java/datameer/awstasks/ant/ec2/AbstractEc2Task.java [hunk 2]
```diff
@@ -51,6 +51,7 @@ private AmazonEC2 createEc2() {
         if (_region != null && !_region.trim().isEmpty()) {
             ec2Client.setRegion(Region.getRegion(Regions.valueOf(_region.toUpperCase())));
         }
+        LOG.info("connect to region " + _region);
```
Comment (mschaaf) [inline src/main/java/datameer/awstasks/ant/ec2/AbstractEc2Task.java:54]: region can be null would be nice if seperate log message could be used or this one moved into the if statement

File: src/main/java/datameer/awstasks/ant/ec2/Ec2TagTask.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.Map;
+import java.util.HashMap;
+
+public class Ec2TagTask extends AbstractEc2ConnectTask {
+
+    private String _key;
+    private String _value;
+
+    public void setTag(String tag)
+    {
```
Comment (mschaaf) [inline src/main/java/datameer/awstasks/ant/ec2/Ec2TagTask.java:32]: code formatting: open brace on the same line like the method name

File: src/main/java/datameer/awstasks/ant/ec2/Ec2TagTask.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        String[] parts = tag.split("=");
+        _key = parts[0];
+        _value = parts[1];
+    }
+
+    @Override
+    protected void doExecute(AmazonEC2 ec2, InstanceGroup instanceGroup) throws Exception {
+        LOG.info("executing " + getClass().getSimpleName() + " with groupName '" + _groupName + "'");
```
Comment (mschaaf) [inline src/main/java/datameer/awstasks/ant/ec2/Ec2TagTask.java:40]: The task name can be hardcoded and it is still part of the log message from the log4j formatter

**Pull request merged**